<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FUSMainWindowClass</class>
 <widget class="QMainWindow" name="FUSMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1364</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FUSMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>751</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QPushButton" name="readButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>75</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>40</y>
      <width>311</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="Timebase_spinBox">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>10</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Timebase</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Buffer_spinBox">
    <property name="geometry">
     <rect>
      <x>146</x>
      <y>10</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100000000</number>
    </property>
    <property name="value">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>49</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Buffer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>y-axis range</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InitializeButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>75</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Initialize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>75</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Close unit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Range_comboBox">
    <property name="geometry">
     <rect>
      <x>561</x>
      <y>10</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <item>
     <property name="text">
      <string>10 mV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20 mV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50 mV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100 mV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200 mV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>500 mV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10 V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20 V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50 V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100 V</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>10</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Range</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>639</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Trigger Voltage</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TriggerVoltage_lineEdit">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>25</string>
    </property>
    <property name="placeholderText">
     <string>25</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="yaxisRange_lineEdit">
    <property name="geometry">
     <rect>
      <x>487</x>
      <y>10</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>150</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GenerateWaveform_Button">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>480</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Waveform</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Abort_Button">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>480</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Abort</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>520</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="WaveformGenerator_GroupBox">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>50</y>
      <width>221</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="WaveformGenerator_gridLayout">
     <item row="19" column="1">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>PRF (Hz)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Amplitude (mVpp)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="24" column="1">
      <widget class="QSpinBox" name="Length_spinBox">
       <property name="maximum">
        <number>600</number>
       </property>
       <property name="value">
        <number>120</number>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QSpinBox" name="DutyCycle_spinBox">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Pulse Duration (ms)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="20" column="1">
      <widget class="QSpinBox" name="PRF_spinBox">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="CheckDeviceButton">
       <property name="text">
        <string>Check Device</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Duty Cycle (%)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="Frequency_spinBox">
       <property name="maximum">
        <number>2000000</number>
       </property>
       <property name="value">
        <number>500000</number>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="17" column="1">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Frequency (Hz)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="1">
      <widget class="QSpinBox" name="Amplitude_spinBox">
       <property name="maximum">
        <number>150</number>
       </property>
       <property name="value">
        <number>80</number>
       </property>
      </widget>
     </item>
     <item row="23" column="1">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Length (s)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="12" column="1">
      <widget class="QSpinBox" name="PulseDuration_spinBox">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="21" column="1">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="SamplingInterval_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sampling interval = </string>
    </property>
   </widget>
   <widget class="QLabel" name="BlockDuration_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Block duration = </string>
    </property>
   </widget>
   <zorder>WaveformGenerator_GroupBox</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>readButton</zorder>
   <zorder>textEdit</zorder>
   <zorder>Timebase_spinBox</zorder>
   <zorder>label</zorder>
   <zorder>Buffer_spinBox</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>InitializeButton</zorder>
   <zorder>CloseButton</zorder>
   <zorder>Range_comboBox</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>TriggerVoltage_lineEdit</zorder>
   <zorder>yaxisRange_lineEdit</zorder>
   <zorder>GenerateWaveform_Button</zorder>
   <zorder>Abort_Button</zorder>
   <zorder>progressBar</zorder>
   <zorder>SamplingInterval_label</zorder>
   <zorder>BlockDuration_label</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1364</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FUSMainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
